MAHelper.cs

```diff
@@ -15,7 +15,7 @@ internal static class MAHelper
         public static void Print(string message)
         {
             ISettingsProvider settings = new MASettings();
-            if (settings.Debug)
+            if(settings.Debug)
```

Comment:
```suggestion
            if (settings.Debug)
```

I think the space should stay.